docs/sources/trackers/google-tag-manager/quick-start/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Quick start
+sidebar_position: 100
+---
+
+This guide will walk you through the initial setup for Snowplow in Google Tag Manager.
+
+## Adding the Templates
+
+To get started with Snowplow in Google Tag Manager, you will need to add the Snowplow Tag Template and the Snowplow Settings Variable Template to your GTM workspace.
```

Comment:
Should link to the higher-level `/docs/sources/trackers/google-tag-manager/index.md` section which has the actual gallery links.